### DISCUSSION TITLE
Question: What does the overlayshaders thing do

### DISCUSSION THREAD

**Discussion by aflacc** [FunkinCrew/Funkin#1514] [Q&A] [answered]
what do it do tho

**Comment by supern64** [comment]
It seems like this class is completely unused from grepping the code.
![image](https://user-images.githubusercontent.com/41817899/114131329-a6bdcf00-992c-11eb-9818-b1c15442a4d4.png)
This class seems like it was originally intended for some sort of filter in Week 4.
![image](https://user-images.githubusercontent.com/41817899/114131521-f8fef000-992c-11eb-9259-0e40ec60601d.png)
But it's completely commented out.
![image](https://user-images.githubusercontent.com/41817899/114131588-0e741a00-992d-11eb-8093-1135c67d381e.png)

**Comment by aflacc** [comment]
ohh neat so there was a planned shading effect in week 4??

**Comment by supern64** [comment]
Probably. I haven't tried activating it.

**Comment by DotWith** [comment]
This was a unused red effect for week 4. It wouldn't work. :(

**Comment by KitOdin** [comment]
its for week 4, uncommenting the lines for said shader and compiling will only result in a bunch of weird fucking errors

**Comment by ninjamuffin99** [comment, marked as answer]
busted ass shaders that was planned for week 4, i didnt know how shaders worked at the time, so instead we just baked the overlay filter effects into the art assets
